instrumentation/src/javatest/org/oppia/android/instrumentation/BaseTest.kt

```diff
@@ -0,0 +1,78 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.Until
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+
+/** Tests to load Oppia using UI Automator. */
+class BaseTest {
+  private val OPPIA_PACKAGE = "org.oppia.android"
+  private val LAUNCH_TIMEOUT = 5000
```

Comment:
I choose 5 seconds as per the Ui Automator examples I've referred to,
Tried using 1 sec and 2 secs both fail, 3 sec seems to be working.

Reply:
We could also use 30 seconds. Generally, we see 10, 30, and 60 second timeouts, or 5 minutes as the longest possible (since Bazel will fail the test at that point) depending on the context. 30 seems more fitting here since there are a lot of things we may need to wait for on the emulator or device side for the test to actually start.

Reply:
Done, I've also added a seperate timeout (TRANSITION_TIMEOUT) of 5 seconds to wait for the navigation between the activities, I used 5 seconds as minimum expecting that for the failure case the 30sec timeout would be very large for the activites.